hack/label-worker-cnf.sh

```diff
@@ -6,8 +6,24 @@ set -e
 OC_TOOL="${OC_TOOL:-oc}"
 
 # Label 1 worker node
-echo "[INFO]: Labeling 1 worker node with worker-cnf"
-node=$(${OC_TOOL} get nodes --selector='node-role.kubernetes.io/worker' \
-    --selector='!node-role.kubernetes.io/master' -o name | head -1)
 
-${OC_TOOL} label --overwrite $node node-role.kubernetes.io/worker-cnf=""
+function label_worker_cnf() {
```

Comment:
we discussed about making sure hypershift worker nodes have the correct labels after each config (tuned, perfprofile, kubeletconfig...) update because this will cause the reprovisioning of the nodes.
I guess this part is not implemented here yet

Reply:
Here  the initial labeling is done. inside the code itself we relabel when needed